docs/content/en/docs/getting-started/production-environment/snow-getstarted.md

```diff
@@ -209,13 +193,22 @@ Follow these steps if you want to use your initial cluster to create and manage
 1. Check the workload cluster:
 
    You can now use the workload cluster as you would any Kubernetes cluster.
-   Change your credentials to point to the new workload cluster (for example, `mgmt-w01`), then run the test application with:
+   * If your workload cluster was created with `eksctl`,
+     change your credentials to point to the new workload cluster (for example, `w01`), then run the test application with:
 
-   ```bash
-   export CLUSTER_NAME=mgmt-w01
-   export KUBECONFIG=${PWD}/${CLUSTER_NAME}/${CLUSTER_NAME}-eks-a-cluster.kubeconfig
-   kubectl apply -f "https://anywhere.eks.amazonaws.com/manifests/hello-eks-a.yaml"
-   ```
+      ```bash
+      export CLUSTER_NAME=w01
+      export KUBECONFIG=${PWD}/${CLUSTER_NAME}/${CLUSTER_NAME}-eks-a-cluster.kubeconfig
+      kubectl apply -f "https://anywhere.eks.amazonaws.com/manifests/hello-eks-a.yaml"
+      ```
+
+   * If your workload cluster was created with GitOps or Terraform, the kubeconfig for your new cluster is stored as a secret on the management cluster.
+     You can get credentials and run the test application as follows:
+      ```bash
+      kubectl get secret -n eksa-system w01-kubeconfig -o jsonpath=‘{.data.value}' | base64 —decode > w01.kubeconfig
+      export KUBECONFIG=w01.kubeconfig
```

Comment:
It seems like this information is repeated this a lot in the docs. Maybe at some point we should extract the details on the kubeconfig location and how to retrieve it on another page and link to it when needed?

Reply:
Good point! Since accessing the kubeconfig is a key part of managing the new cluster, I added this information in multiple places to ensure that there was no confusion or seemingly missing information for the users. So whether they begin following the docs from the Gitops/Terraform pages or if they are following it from the provider-specific "getting started" pages, they would ideally have all of the information needed to use this API to manage their clusters. I would rather over-communicate than under-communicate.

Would definitely be open to ideas on a central location for this information!

Reply:
I agree. I don't mind the text appearing multiple times, but it would be best to have it pulled somehow from the same place each time it is used. I'll look into ways to do that.